osm-seed/values.yaml

```diff
@@ -286,15 +286,18 @@ tilerDb:
     POSTGRES_DB_MAINTENANCE_WORK_MEM: 64MB
     POSTGRES_DB_EFFECTIVE_CACHE_SIZE: 4GB
   persistenceDisk:
-    enabled: false
+    enabled: true
     accessMode: ReadWriteOnce
     mountPath: /var/lib/postgresql/data
     subPath: postgresql-d
-    # In case cloudProvider: aws
-    # AWS_ElasticBlockStore_volumeID : vol-1234bcd
-    # AWS_ElasticBlockStore_size: 500Gi
-    # In case cloudProvider: gcp
-    GCP_gcePersistentDisk_pdName: osmseed-disk-tiler-db-v2
+    # Minikube
+    local_volume_hostPath: /mnt/tiler-db-data
```

Comment:
I imagine the user will need to customize these values before they can deploy? also, wondering if we should stick to regular camel case here like the other variables, so for the sake of consistency, call it `localVolumeHostPath` ? Would leave that decision to you tho, @Rub21

Reply:
@batpad  Good, that makes sense, i will rename rename the variable  in  camel case format: `localVolumeHostPath` but  later we may need to rename the following vars too 👇 , in order to standardize the vars.

```sh
    # AWS
    AWS_ElasticBlockStore_volumeID: vol-1234bcd
    AWS_ElasticBlockStore_size: 50Gi
    # GCP
    GCP_gcePersistentDisk_pdName: osmseed-disk-imposm-v1
    GCP_gcePersistentDisk_size: 50Gi
```
Renaming those variable may need their own ticket , to work later. because it need to be renamed in ohm-deployment repo too.